src/main/java/seedu/fast/logic/parser/FindCommandParser.java

```diff
@@ -24,10 +26,33 @@ public FindCommand parse(String args) throws ParseException {
             throw new ParseException(
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
+        if (trimmedArgs.startsWith(PriorityTag.PRIORITY_TAG_PREFIX)) {
+            trimmedArgs = trimmedArgs.substring(
```

Comment:
I think that the naming scheme here can be improved. I for example after getting the substring, it can be called "extractedPriorityTag" or "tokenizedArgs". This better conveys the meaning of the code rather than reusing trimmedArgs.